package.json

```diff
@@ -1,45 +1,50 @@
 {
-  "type": "commonjs",
-  "scripts": {
-    "test": "tsc && npm run jasmine",
-    "test-parallel": "tsc && npm run jasmine-parallel",
-    "jasmine": "node --enable-source-maps node_modules/jasmine/bin/jasmine --config=./jasmine.json --random=false",
-    "jasmine-parallel": "node --enable-source-maps node_modules/jasmine/bin/jasmine --config=./jasmine.json --parallel=4 --random=true",
-    "lint": "eslint --quiet",
-    "lint-verbose": "eslint",
-    "get-cards": "node ./bin/fetchdata.js"
-  },
-  "dependencies": {
-    "@eslint/eslintrc": "^3.1.0",
-    "@eslint/js": "^9.8.0",
-    "@stylistic/eslint-plugin-js": "^2.6.1",
-    "@types/eslint__js": "^8.42.3",
-    "@types/jasmine": "^5.1.4",
-    "@types/underscore": "^1.11.15",
-    "axios": "^1.7.2",
-    "axios-retry": "^4.5.0",
-    "cli-progress": "^3.12.0",
-    "eslint": "^9.8.0",
-    "eslint-plugin-import": "^2.31.0",
-    "eslint-plugin-jasmine": "^4.2.0",
-    "events": "^3.3.0",
-    "fs": "^0.0.1-security",
-    "globals": "^15.9.0",
-    "jasmine": "^5.1.0",
-    "jasmine-spec-reporter": "^7.0.0",
-    "jasmine-terminal-reporter": "^1.0.3",
-    "jasmine-ts": "^0.4.0",
-    "mkdirp": "^3.0.1",
-    "node-ts": "^6.1.0",
-    "path": "^0.12.7",
-    "ts-node": "^10.9.2",
-    "typescript": "^5.5.4",
-    "typescript-eslint": "^8.0.1",
-    "underscore": "^1.13.6",
-    "uuid": "^10.0.0",
-    "zod": "^3.23.8"
-  },
-  "devDependencies": {
-    "@types/node": "^20.14.10"
-  }
-}
+    "type": "commonjs",
```

Comment:
Not sure, I did add a package for copying files around so it perhaps shifted something. ESLint also started auto-formatting some of the files when I open/save them, so it might be the culprit.